### PR TITLE
doc: update usage of some suite commands

### PR DESCRIPTION
**Pull request by matthewrmshin**
@arjclark please review. (1 review enough?)

### CODE REVIEW

File: doc/rose-rug-files.html [hunk 1]
```diff
@@ -444,7 +444,9 @@ <h3 id="rose-config">rose config</h3>
     <div class="slide">
       <h3 class="alwayshidden">rose config --file</h3>
 
-      <p>rose config also works for the other Rose formats. For example:</p>
+      <p>rose config also works for other Rose configuration files (and any
```
Comment (arjclark) [inline doc/rose-rug-files.html]: Could you pop this in ```<kbd>``` tags like the other commands (I know it wasn't originally but while you're in here we may as well change it)

File: doc/rose-rug-files.html [hunk 2]
```diff
@@ -522,17 +524,27 @@ <h3 class="alwayshidden">Other Useful Commands</h3>
...
-
-        <li><kbd>rose suite-clean</kbd>: remove items created by <code>rose
-        suite-run</code> for the suite in <var>$PWD</var> and those on remote
-        job hosts.</li>
+        <li><kbd>rose suite-scan</kbd> or <kbd>cylc scan</kbd>: list your
+        running suites.</li>
+
+        <li><kbd>cylc gscan</kbd>: GTK version of <kbd>cylc scan</kbd>.</li>
+
+        <li><kbd>rose suite-gcontrol [--name=NAME]</kbd> or <kbd>cylc gui
```
Comment (arjclark) [inline doc/rose-rug-files.html]: Not sure about telling people to run cylc gui like this as it needs the --host option to make it run on the correct server - if its not provided you (currently) get a "connection failed" dialog on opening of the cylc gui - or has one of your recent cylc changes altered this behaviour?

File: doc/rose-rug-files.html [hunk 3]
```diff
@@ -522,17 +524,27 @@ <h3 class="alwayshidden">Other Useful Commands</h3>
...
+        NAME</kbd>: launch the <code>cylc gui</code> for a suite called
+        <var>NAME</var>. The rose wrapper uses the base name of <var>$PWD</var>
+        as <var>NAME</var> in the absence of the <kbd>--name=NAME</kbd>
+        option.</li>
+
+        <li><kbd>rose suite-shutdown [--name=NAME]</kbd> or <kbd>cylc stop
```
Comment (arjclark) [inline doc/rose-rug-files.html]: Same comment as above re. needing --host= in the current version of cylc.

**Comment by arjclark** [comment]
@matthewrmshin - couple of comments above. Primarily, do these additions of the cylc commands as written reflect a recent change in cylc?

File: doc/rose-rug-files.html (hunk 2 above)
Comment (matthewrmshin) [inline doc/rose-rug-files.html]: Not since cylc-6.8.0.

File: doc/rose-rug-files.html (hunk 3 above)
Comment (matthewrmshin) [inline doc/rose-rug-files.html]: Not since cylc-6.8.0.

**Comment by matthewrmshin** [comment]
Please note that `--host=HOST` is no longer necessary with cylc-6.8.0.

File: doc/rose-rug-files.html (hunk 1 above)
Comment (matthewrmshin) [inline doc/rose-rug-files.html]: Done.

**Comment by arjclark** [comment]
Cool, confirmed. Looks ok to me.

**Comment by arjclark** [comment]
For reference, my above comments came out of testing with an earlier version of cylc (don't ask). It'd be a concern if we **didn't** recommend the equivalent latest version of cylc to be used with the Rose release but I think this is fine and ties in with our long term goals of moving Rose functionality into cylc.

**Pull request merged**